torch/csrc/lazy/backend/backend_interface.h

```diff
@@ -132,7 +132,7 @@ class TORCH_API BackendRegistrar {
   BackendRegistrar(const BackendImplInterface* backend_impl_interface);
 };
 
-TORCH_API bool hasBackend();
+bool hasBackend();
```

Comment:
why is this removed?

Reply:
hmm, not sure. I don't remember removing it. I think it was just probably missed in a rebase. Will revert now